src/improvements/doc/TASK_CREATION_GUIDE.md

```diff
@@ -0,0 +1,196 @@
+# Task Creation Guide
+
+This guide walks through the process of creating new tasks using existing templates, including network-specific considerations and best practices.
+
+## Quick Start
+
+To scaffold a new task using an existing template:
+
+```bash
+cd src/improvements/
+just new task
+```
+
+Follow the prompts from the justfile to create a new task directory with pre-populated README.md and config.toml files.
+
+## Directory Structure
+
+Tasks are organized by network in the following structure:
+
+```
+src/improvements/tasks/
+├── eth/           # Ethereum mainnet tasks
+├── sep/           # Sepolia testnet tasks
+├── oeth/          # Optimism Ethereum tasks
+├── opsep/         # Optimism Sepolia tasks
+└── common/        # Shared resources and documentation
+```
+
+### Network-Specific Organization
+
+1. Ethereum Mainnet (tasks/eth/)
+   - Production tasks
+   - Strict security requirements
+   - Full testing required
+   - Example: eth/001-security-council-phase-0/
+
+2. Sepolia Testnet (tasks/sep/)
+   - Testing and validation
+   - Development tasks
+   - Example: sep/001-op-extended-pause/
+
+3. Optimism Networks
+   - tasks/oeth/: Optimism Ethereum tasks
+   - tasks/opsep/: Optimism Sepolia tasks
+   - Network-specific configurations required
+
+### Task Naming Convention
+
+Tasks follow a numbered sequence within each network directory:
```

Comment:
This isn't correct. We want new tasks to follow lexicographical ordering. If a user is interested in the actual lexicographical ordering for a networks tasks they can run: `just task ls`. When creating a new task with `just new task` it tries to get the user to create the next task in lexicographical order.